api-specificatie/Bevraging-Historie/openapi.yaml

```diff
@@ -254,29 +243,36 @@ paths:
             type: string
             maxLength: 9
             minLength: 9
+        - in: header
+          name: api-version
+          description: "Bevat de versie van de aan te roepen API conform Landelijke API-strategie."
+          required: false
+          schema:
+            type: string
+            example: 1.0.1
         - in: query
           name: fields
-          description: "Geeft de mogelijkheid de inhoud van de body van het antwoord naar behoefte aan te passen. Bevat een door komma's gescheiden lijst van veldennamen. Als niet-bestaande veldnamen worden meegegeven wordt een 400 Bad Request teruggegeven. Wanneer de parameter fields niet is opgenomen, worden alle gedefinieerde velden die een waarde hebben teruggegeven. Zie [functionele specificaties](https://github.com/VNG-Realisatie/Bevragingen-ingeschreven-personen/blob/master/features/fields.feature)"
+          description: "Geeft de mogelijkheid de inhoud van de body van het antwoord naar behoefte aan te passen. Bevat een door komma's gescheiden lijst van veldennamen. Als niet-bestaande veldnamen worden meegegeven wordt een 400 Bad Request teruggegeven. Wanneer de parameter fields niet is opgenomen, worden alle gedefinieerde velden die een waarde hebben teruggegeven. Zie functionele specificaties)"
           required: false
           schema:
             type: string
         - in: query
           name: geldigop
-          description: "De peildatum van de periode waarover de partnerhistorie wordt opgevraagd. Het resultaat bevat de partnerhistorie waarvan de datumIngangGeldigheid op of voor de peildatum ligt en de datumEindeGeldigheid op of na de peildatum ligt."
+          description: "De peildatum van de periode waarover de partnerhistorie wordt opgevraagd. Het resultaat bevat de partners op het moment van de peildatum."
           required: false
           schema:
             type: string
             format: date
         - in: query
           name: geldigtot
-          description: "De einddatum van de periode waarover de partnerhistorie wordt opgevraagd. Het resultaat bevat de partnerhistorie waarvan de datumEindeGeldigheid op of voor datumTotEnMet ligt."
+          description: "De einddatum van de periode waarover de partnerhistorie wordt opgevraagd. Het resultaat bevat de partners waarvan de datumEindeGeldigheid op of voor geldigTot ligt."
```

Comment:
Niet op **en** voor toch? Alleen voor. Als het op de Datumeindegeldigheid is dan is die niet meer geldig toch?

Reply:
Analoog aan een eerder voorstel heb ik deze zin nu verwijderd uit de description om alle implementatiedetails buiten de OS te houden.